packages/playground/css-codesplit-cjs/main.css

```diff
@@ -0,0 +1,3 @@
+h1 { 
```

Comment:
**nitpick:** I know it's just a playground file, but seems you need to format this file manually, cause it's not covered by a git-hook

There is a trailing whitespace and we should use an `eol` for such files

Reply:
Copied from the `css-codesplit` playground, might need to to the overall formating later.

Reply:
Wasn't catched by https://github.com/vitejs/vite/pull/2768? :disappointed:  
We really need to setup a global hook for every file, not only js, ts and html. Prettier can literally format almost everything.